types/repo/AnonymousMessageRepo/AnonymousChannelsData.ts

```diff
@@ -86,5 +90,7 @@ export interface AnonymousChannelData {
   messages: Message[];
   targetName: string;
   targetImage: string;
+  myUserId?: string;
+  unreadCount?: number;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `myUserId` and `unreadCount` fields have been added to the `Conversation` interface. As with the previous comments, ensure these new fields are properly handled wherever this interface is used.

```diff
+  myUserId?: string;
+  unreadCount?: number;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->